src/mlpack/core/data/split_data.hpp

```diff
@@ -169,6 +169,7 @@ void StratifiedSplit(const arma::Mat<T>& input,
  *                testData, trainLabel, testLabel, 0.3);
  * @endcode
  *
+ * @tparam LabelsType Type of input labels. It must be arma::Mat or arma::row.
```

Comment:
Might as well document `T` too?

Reply:
Sure. I will do it. Somehow, I missed it earlier :)